### PR TITLE
PHPUnit support 5

### PR DESCRIPTION
**Pull request by kokspflanze**
phpunit getMock removed and added getMockBuilder

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
         "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
         "zendframework/zend-session": "^2.6.2",
         "zendframework/zend-uri": "^2.5",
-        "fabpot/php-cs-fixer": "1.7.*",
-        "phpunit/PHPUnit": "^4.0"
+        "friendsofphp/php-cs-fixer": "1.7.*",
```
Comment (kokspflanze) [inline composer.json]: removed it from this PR

**Comment by kokspflanze** [comment]
@adamlundrigan @weierophinney should i close it and recreate it with php5.6+ and phpunit6?

**Comment by Xerkus** [comment]
@kokspflanze you can update this PR, or rebase to pick relevant changes and force push

**Comment by Xerkus** [comment]
This PR is superseded by #151

**Pull request closed without merging**